octodns/zone.py

```diff
@@ -110,10 +110,29 @@ def changes(self, desired, target):
         for record in filter(_is_eligible, self.records):
             if record.ignored:
                 continue
+            elif len(record.included) > 0 and \
+                    target.__class__.__name__ not in record.included:
```

Comment:
What do you think about using `target.id`?

In cases where you have multiple instances of a single provider you'd be able to control which one(s) something should be included or excluded by with finer granularity. You could always get back to it being a provider-based thing by listing multiple ids (all the ids for the class.) I'd imagine most configs will just have one instance of a given provider class, but at least internally with PowerDNS we have many. It also seems a little more consistent to talk in terms of `id` since that's what's used in `sources` and `targets`.

Reply:
Actually I did not like `target.__class__.__name__` but wasn't aware that we could use simply class.id :-). Thanks for the review, update on its way!